Makefile

```diff
@@ -45,21 +47,24 @@ test:
 	$(GOTEST) test -coverprofile coverage.out -race -v ./...
 
 .PHONY: build
-build: dev/api dev/gitpods dev/storage dev/ui
+build: dev/api dev/gitpods-dev dev/storage dev/ui
 
-dev/api: cmd/api pkg
+dev/api: cmd/api $(GO_PKG_FILES)
 	$(GO) build -v -ldflags '-w -extldflags '-static'' -o ./dev/api ./cmd/api
 
-dev/gitpods-dev: cmd/gitpods-dev pkg
+dev/gitpods-dev: cmd/gitpods-dev $(GO_PKG_FILES)
 	$(GO) build -v -ldflags '-w -extldflags '-static'' -o ./dev/gitpods-dev ./cmd/gitpods-dev
 
-dev/storage: cmd/storage pkg
+dev/storage: cmd/storage $(GO_PKG_FILES)
 	$(GO) build -v -ldflags '-w -extldflags '-static'' -o ./dev/storage ./cmd/storage
 
-dev/ui: cmd/ui pkg dev/packr
+dev/ui: cmd/ui $(GO_PKG_FILES) ui/build dev/packr
 	./dev/packr
 	$(GO) build -v -ldflags '-w -extldflags '-static'' -o ./dev/ui ./cmd/ui
 
+ui/build: $(DARTFILES)
```

Comment:
Have you tried running this command twice? It should only compile with `webdev build` on the first time, right? Just want to make sure! :slightly_smiling_face:

Reply:
Yes, `make: 'ui/build' is up to date.` :slightly_smiling_face:

Reply:
(it's all cached in `~/.pub-cache`, hence the reduced output)
```
$ make ui/build
cd ui && webdev build
Creating build script, took 13395ms
[INFO] Initializing inputs
[INFO] Reading cached asset graph...
[INFO] Reading cached asset graph completed, took 710ms
[INFO] Checking for updates since last build...
[INFO] Checking for updates since last build completed, took 876ms
[INFO] Running build...
[INFO] Running build completed, took 469ms
[INFO] Caching finalized dependency graph...
[INFO] Caching finalized dependency graph completed, took 354ms
[INFO] Creating merged output dir `build`...
[INFO] Creating merged output dir `build` completed, took 586ms
[INFO] Writing asset manifest...
[INFO] Writing asset manifest completed, took 3ms
[INFO] Succeeded after 1.5s with 0 outputs (0 actions)

$ make ui/build
make: 'ui/build' is up to date.
```

Reply:
Only thing is, it only rebuilds when dart-files are modified, not if css/html is... Not sure if that's the intended behaviour :thinking:

Reply:
Very nice. Yeah, not sure about this yet. For now I think only Dart files are fine. HTML and CSS won't be changed that much any way and if so people will use the `webdev serve` command in the background.